test/flycheck-test.el

```diff
@@ -4124,7 +4124,8 @@ Why not:
           :checker ruby-jruby))))
 
 (flycheck-ert-def-checker-test rust-cargo rust warning
-  (let ((flycheck-rust-crate-type "bin"))
+  (let ((flycheck-rust-crate-type "bin")
+        (flycheck-rust-binary-name "flycheck"))
```

Comment:
We don't.  I initially passed `--bin` unconditionally, which required a binary name.  But then I changed the test in the checker to fallback to the previous behavior of not specifying any binary name, and forgot to remove this line.